### ISSUE TITLE
Space around operator false positive

### ISSUE THREAD

**Issue by bgriffith** [sasstools/sass-lint#394]
The space-around-operator rule recently added to dev branch has an issue with valid css using negative values.

```css
.foo { margin: -15px -15px; }
``` 

Sass 1.3.2, node 5.0.0
